packs/reamaze/actions/article_search.py

```diff
@@ -0,0 +1,32 @@
+from lib.actions import BaseAction
+from dotted.utils import dot
+from fuzzywuzzy import fuzz
+import json
+
+
+class ArticleSearch(BaseAction):
+    def run(self, query, include_body=False, limit=5):
+        response = self._api_get('/articles')
```

Comment:
At some point this could return a large number of articles. I guess we better watch for native search or pagination support.